### PR TITLE
Correcting some style issues

### PR DESCRIPTION
**Pull request by kalexand-rh**
This PR fixes a few style issues across the repo. It removes "please," swaps "once" for "after," and corrects the syntax for a few optional steps.

Preview here: https://deploy-preview-35048--osdocs.netlify.app/openshift-enterprise/latest/nodes/jobs/nodes-nodes-jobs.html#nodes-nodes-jobs-creating_nodes-nodes-jobs

4.8+

### CODE REVIEW

File: welcome/index.adoc [hunk 1]
```diff
@@ -239,7 +239,7 @@ cluster can be performed by {product-title} cluster administrators. As a
 
 ////
 - **xref:../operators/understanding/olm-what-operators-are.adoc#olm-what-operators-are[Manage Operators]**: Lists of Red Hat, ISV, and community Operators can
-be reviewed by cluster administrators and xref:../operators/admin/olm-adding-operators-to-cluster.adoc#olm-adding-operators-to-a-cluster[installed on their clusters]. Once installed, you can xref:../operators/user/olm-creating-apps-from-installed-operators.adoc#olm-creating-apps-from-installed-operators[run], upgrade, back up or otherwise manage the Operator on your cluster (based on what the Operator is designed to do).
+be reviewed by cluster administrators and xref:../operators/admin/olm-adding-operators-to-cluster.adoc#olm-adding-operators-to-a-cluster[installed on their clusters]. After installed, you can xref:../operators/user/olm-creating-apps-from-installed-operators.adoc#olm-creating-apps-from-installed-operators[run], upgrade, back up or otherwise manage the Operator on your cluster (based on what the Operator is designed to do).
```
Comment (bergerhoffer) [inline welcome/index.adoc:242]: Same comment about additional words

**Review by bergerhoffer — COMMENTED** [review]
Thanks for submitting these fixes!

Just a reminder to squash commits before merging. I'd also recommend trying to cherry pick these changes back farther than 4.8. I know it's likely that you'll run into conflicts that you'll have to resolve, but it's better to do them now rather than have someone else run into the conflict in their PR later.

File: modules/nw-sriov-dpdk-example-intel.adoc [hunk 2]
```diff
@@ -39,7 +39,7 @@ spec:
 +
 [NOTE]
 =====
-Please refer to the `Configuring SR-IOV network devices` section for a detailed explanation on each option in `SriovNetworkNodePolicy`.
+Refer to the `Configuring SR-IOV network devices` section for a detailed explanation on each option in `SriovNetworkNodePolicy`.
```
[the remark this answers is not in the document]
Comment (kalexand-rh) [reply modules/nw-sriov-dpdk-example-intel.adoc:42]: I'll go ahead and fix the rest of the instance of "Refer to," too. Thanks!

**Pull request merged**